contracts/NewfiAdvisor.sol

```diff
@@ -162,15 +162,14 @@ contract NewfiAdvisor is ReentrancyGuardUpgradeSafe, ProxyFactory, Helper {
     {
         Advisor storage advisor = advisorInfo[_advisor];
         address stablePool = advisor.stablePool;
-        //        uint256 mstablePoolInvestmentValue = SavingsContract(savingContract)
-        //            .creditBalances(stablePoolProxy)
-        //            .mul(SavingsContract(savingContract).exchangeRate());
-        //        // whitelisted vault addresses for now
-        //        uint256 yearnPoolInvestmentValue = YearnVault(getUSDCVault())
-        //            .balanceOf(advisor.volatilePool)
-        //            .mul(YearnVault(getUSDCVault()).getPricePerFullShare());
-        //        return mstablePoolInvestmentValue.add(yearnPoolInvestmentValue);
-        return 1;
+        uint256 mstablePoolInvestmentValue = SavingsContract(savingContract)
```

Comment:
okay fine

Reply:
I think this was just an accident, pretty sure I pushed it up uncommented?